packages/expo-clipboard/ios/ClipboardModule.swift

```diff
@@ -1,22 +1,81 @@
 // Copyright 2018-present 650 Industries. All rights reserved.
 
 import ExpoModulesCore
+import UIKit
 
 let onClipboardChanged = "onClipboardChanged"
 
 public class ClipboardModule: Module {
   public func definition() -> ModuleDefinition {
     name("ExpoClipboard")
 
+    // MARK: Strings
+
     function("getStringAsync") { () -> String in
       return UIPasteboard.general.string ?? ""
     }
 
-    function("setStringAsync") { (content: String?) in
+    function("setStringAsync") { (content: String?) -> Bool in
       UIPasteboard.general.string = content ?? ""
       return true
     }
 
+    function("hasStringAsync") { () -> Bool in
+      return UIPasteboard.general.hasStrings
+    }
+
+    // MARK: URLs
+
+    function("getUrlAsync") { () -> String? in
+      return UIPasteboard.general.url?.absoluteString
+    }
+
+    function("setUrlAsync") { (content: String) in
+      guard let url = URL(string: content) else {
+        throw InvalidUrlException(content)
+      }
+      UIPasteboard.general.url = url
+    }
```

Comment:
```suggestion
    function("setUrlAsync") { (url: URL) in
      UIPasteboard.general.url = url
    }
```
This should work too 😉 You can find the conversion here https://github.com/expo/expo/blob/b51a1b6bffb689b90aa1c997d6568acdb62d0318/packages/expo-modules-core/ios/Swift/Arguments/Convertibles.swift#L14